signing_service/signing_service/signing_service.py

```diff
@@ -153,7 +157,6 @@ def _attempt_reconnection(self) -> None:
         self._increase_delay()
         self.reconnection_counter += 1
         if self.reconnection_counter > self.maximum_reconnection_attempts:
-            crash_logger.error("Maximum reconnection attempts exceeded.")
             raise SigningServiceMaximumReconnectionAttemptsExceeded
```

Comment:
Why not log it here?

Reply:
It could be either here or in the place where it is currently (in the mail loop). I have decided to put it there because every `except` block there has a logger usage and for somebody reading the code logically from top to bottom (from main loop down to the more specific functions) it might be more describing. But I have not any better argument for that. Definitely it should not be logged twice.

Reply:
OK